src/components/tooltip/Marker.js

```diff
@@ -545,6 +545,17 @@ function getMarkerDetails(
         );
         break;
       }
+      case 'Preference': {
+        tooltipDetails = (
+          <TooltipDetails>
+            <TooltipDetail label="Pref Name">{data.prefName}</TooltipDetail>
+            <TooltipDetail label="Pref Kind">{data.prefKind}</TooltipDetail>
+            <TooltipDetail label="Pref Type">{data.prefType}</TooltipDetail>
+            <TooltipDetail label="Pref Value">{data.prefValue}</TooltipDetail>
```

Comment:
Up to you as you're the primary customer :) but maybe we can emove the prefix `Pref` in labels. What do you think?

Reply:
Great idea! I think that I will definitely make this change. Thanks.